libs/ts-rest/express/src/lib/ts-rest-express.ts

```diff
@@ -171,7 +170,7 @@ const transformAppRouteQueryImplementation = (
 
       return res.status(statusCode).json(result.body);
     } catch (e) {
-      return next(e);
+      return next?.(e);
```

Comment:
There used to be a problem with the typing of the `next` function because it was grabbing incorrect typing from `express-static-serve-core` instead of `@types/express-static-serve-core`, but that should be the case now.

As you can see [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-serve-static-core/index.d.ts#L72) `next` is not optional, so there should be no need for the optional chaining operator

Reply:
Ahh I see.., My bad, so It was because my `node_modules` wasn't up to date. I haven't done `pnpm install` again after syncing this [commit](https://github.com/ts-rest/ts-rest/commit/41a5fbbe45dc9563ebbf554ad0677b07997dab59#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L71).

Will revert it back in my next PR, thanks for the review!